lib/strMap.ts

```diff
@@ -0,0 +1,70 @@
+/// <reference path='../typings/tsd.d.ts' />
+
+export = StrMap;
```

Comment:
What is the motivation for `StrMap`?  I would just use an `Object` as a dictionary and save having an alternative implementation.  ES6 maps are on their way soon hopefully with the next release of NodeJS.